recipes/symengine/bld.bat

```diff
@@ -0,0 +1,19 @@
+mkdir build
+cd build
+
+cmake ^
+    -G "%CMAKE_GENERATOR%" ^
```

Comment:
`set "CMAKE_GENERATOR=Visual Studio 14 2015"` was changed to
`set CMAKE_GENERATOR="Visual Studio 14 2015"`

Using `cmake -G "%CMAKE_GENERATOR%"` worked earlier, now it should be changed to `cmake -G %CMAKE_GENERATOR%`.